Dashboard/src/components/Infrastructure.js

```diff
@@ -48,8 +48,8 @@ class Infrastructure extends Container {
                       key={"workstation-" + i}
                       className="infra-box"
                     >
-                      <img className="ip-icon" src={server.Status === "up" ? zap : zapOff} alt={"IP icon"} />
-                      <span> {server.Host} </span>
+                      <img className="ip-icon" src={server.status === "up" ? zap : zapOff} alt={"IP icon"} />
+                      <span> {server.host} </span>
```

Comment:
I see a lot of repetition, this block is repeated 3 times & looks almost identical:
```
<div className="infra-wrapper">
   <div className="infra-column">
         <div className="infra-box"><h3>{Title}</h3></div>
     ...
       </div>
 </div>
```
I think it could be moved to a function outside of render & you could pass dns, workstations & servers to it

Reply:
I didn't touch the structure of the frontend since that is covered in #65. The only changes I made here were to line up with the backend changes to the data structure.